client/src/components/ProfileNavigation.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const InfoNav = styled.ul`
+  width: 280px;
+  display: flex;
+  flex-direction: column;
+`;
+
+const InfoEntry = styled.li`
+  height: 40px;
+  width: 100%;
+  list-style: none;
+  background: ${({ item, theme }) => (item % 2 === 0 ? theme.colors.active : "")};
+  color: ${({ theme }) => {
+    return theme.colors.darkfont;
+  }};
```

Comment:
usually you use `color: ${({ theme }) => theme.colors.darkfont};`, why not here?